### PR TITLE
Feature/xml export

### PR DESCRIPTION
**Pull request by rubenwiersma**
Added XML export functionality and integration into front-end.

### CODE REVIEW

File: src/main/java/nl/tudelft/thefirstorder/service/util/PDFExportUtil.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ private static void addContent(Document document, Project project) throws Docume
             cameratable.addCell(camera.getY() + "");
             CameraAction action = cue.getCameraAction();
             actiontable.addCell(action.getName());
-//            actiontable.addCell(cue.getTimePoint().getDuration() + "");
+            // actiontable.addCell(cue.getTimePoint().getDuration() + "");
```
Comment (rubenwiersma) [inline src/main/java/nl/tudelft/thefirstorder/service/util/PDFExportUtil.java]: Shouldn't we remove this rule?

File: src/main/java/nl/tudelft/thefirstorder/service/util/XMLExportUtil.java [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+     */
+    private XMLExportUtil() {
+    }
+
+    /**
+     * Export the project.
+     * @param project the project
+     * @return A resource to be downloaded
+     */
+    public static Resource export(Project project) {
```
Comment (rubenwiersma) [inline src/main/java/nl/tudelft/thefirstorder/service/util/XMLExportUtil.java]: Name of the export function is inconsistent with PDF export function.

File: src/main/java/nl/tudelft/thefirstorder/web/rest/ProjectResource.java [hunk 3]
```diff
@@ -181,4 +182,41 @@
...
+        }
+
+        log.debug("Export Project {} to XML.", id);
+        Project currentProject = project.get();
+        Resource resource = XMLExportUtil.exportProjectToXML(currentProject);
+
+        return ResponseEntity.ok()
+            .headers(headers)
+            .contentLength(resource.contentLength())
+            .contentType(MediaType.parseMediaType("application/octet-stream"))
```
Comment (hungnguyen94) [inline src/main/java/nl/tudelft/thefirstorder/web/rest/ProjectResource.java]: I'm not sure if this contentType is correct for a XML.

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -11,7 +11,7 @@
     <!-- bower:css -->
```
Comment (hungnguyen94) [inline src/main/webapp/index.html]: Don't commit index.html.

**Comment by hungnguyen94** [comment]
Please don't commit index.html. It's unnecessary.

File: src/main/java/nl/tudelft/thefirstorder/web/rest/ProjectResource.java (hunk 3 above)
Comment (rubenwiersma) [inline src/main/java/nl/tudelft/thefirstorder/web/rest/ProjectResource.java]: Updated it to application/xml

**Pull request merged**